src/Akeneo/Platform/Installer/back/src/Domain/Query/FindTablesInterface.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * @copyright 2023 Akeneo SAS (https://www.akeneo.com)
+ * @license https://opensource.org/licenses/osl-3.0.php Open Software License (OSL 3.0)
+ */
+
+namespace Akeneo\Platform\Installer\Domain\Query;
+
+interface FindTablesInterface
```

Comment:
This name is not very relevant because we don't understand which tables will be finded/returned. What do you think about something like this:

```suggestion
interface ListTablesToPurgeInterface
```
OR
```suggestion
interface FindTablesInterface
{
    /**
     * @return string[]
     */
    public function toPurge(): array;
}
```

Reply:
I don't know because the logic to filter some tables are in the application layer this function return all tables. We should decide if we move the logic to filter into the Infrastructure layer or we should keep it in the application layer